### DISCUSSION TITLE
Trabalho Remoto — Confiança e Disponibilidade

### DISCUSSION THREAD

**Discussion by renanlara** [frontendbr/forum#1099] [Discussão]
Bom dia galerinha!

Pessoal, eu tenho observado ultimamente que a dificuldade de encontrar trabalho remoto está aumentando. No período de 1 ano, eu tenho conversado com diversas agências a respeito e a dificuldade para a entrega de projetos/tarefas com profissionais remotos aumenta.

As empresas que repassavam projetos para terceiros, hoje só contrata pessoas para trabalhar alocado no escritório, outras, já tiveram experiências por muitos anos com equipes remotas e o resultado nunca foi satisfatório para a empresa em si.

As empresas alegam que os profissionais demoram para entregar os projetos, que a comunicação com esses desenvolvedores é limitada, pois só respondem quando realmente podem e que para o profissional entender de fato o que é necessário realizar na tarefa/projeto é grande e que com uma pessoa presente alocado isso não aconteceria.

Eu gostaria de saber qual é a opinião de vocês:
- Os profissionais trabalham durante o dia alocado e atendem as agências a noite/madrugada quando podem?
- Os profissionais trabalham home office mas se enchem de freelas e não conseguem atender as agências?
- A falta de responsabilidade é grande o suficiente que o profissional não está nem aí para a entrega do projeto?

A confiança das agências/empresas de desenvolvimento pelo visto tende a cair. Algumas até abrem a oportunidade de trabalhar 1 ou 2 dias remotos. Mas mesmo assim não é tão simples de encontrar.

Um outro detalhe, é que as vagas para trabalho remoto a maioria é para o pessoal que é Full Stack JavaScript, aqui mesmo no repositório de vagas já vi muito isso.

**Obs; Me refiro a vagas nacionais, nem todos estão preparados para atender empresas fora do Brasil.**

**Comment by luizbills** [comment]
Essa história é muito mais antiga do que parece. Quando o assunto é agência, na maioria das vezes pagam muito mal tanto para quem é freela quando para quem é alocado. E agência vira e mexe rola o famoso *job urgente*, aquele job que o cara vem te incomodar fora do horário comercial ou fica te pedindo para dar prioridade para ele pois ele precisa entregar o job para o cliente dele (quase sempre a palavra *amanhã* está inclusa). Eu como freelancer, acho um saco... o cara deixa para última hora e ainda acha ruim quando vc não pode *largar seus outros clientes para atender ele*. 

Até hoje, felizmente, só me ligaram bem tarde uma vez. Na ocasião fiquei bem puto, mas me segurei e ouvi o cliente (e não era nada grave que não podia deixar até o proximo dia).

Também ouço muito dos meus clientes que o programador sumiu, parou de atender do nada, etc. Então há sim uma falta de responsabilidade dos profissionais de desenvolvimento. Mas tenho quase certeza que são profissionais bem ruins, que cobravam ou recebiam pouco. 

Mas não dá muito para escolher qual lado está errado e qual está certo. Eu mesmo faço apenas serviços especializados, fujo dessa onda de *full-stack* porque não compensa saber uma duzia de tecnologias (e não dominar nenhuma) e ainda receber pouco. Muita dor de cabeça e muito estresse para pouca recompensa.

**Comment by wmartins** [comment]
Vou falar do lado "de quem contrata" (entre aspas pois foi a minha antiga empresa que contratou, não eu). No caso, trabalhei em um projeto em que um pedaço do projeto foi terceirizado. Essa terceirização foi feita com uma empresa que trabalhava de forma remota. Então, acho que é relevante para a discussão.

No nosso cenário, funcionou bem e foi satisfatório. Acho que isso aconteceu pois essa empresa terceirizada é bem "madura". Eles se organizaram de uma forma boa em relação às demandas e desde o primeiro momento estabelecemos todas as espectativas em relação à prazos, escopo e etc. Além disso, um ponto crucial foi **saber o que queríamos**. Isso facilita a vida de todo mundo.

Acho que o maior problema que pode acontecer é a questão da comunicação. Por vezes (e isso já aconteceu no meu cenário) a empresa que contrata **não** está preparada para trabalhar com pessoas que estão remotas. As empresas acreditam que estão, mas, na prática, acabam fazendo coisas que não contribuem.

Por exemplo, em alguns momentos o projeto continha algumas indefinições e incertezas, que, ao invés de serem discutidas de uma melhor forma para diminuir as incertezas, eram simplesmente passadas para serem feitas pois "estávamos pagando por recursos". No fim, o pessoal remoto entregava o que tinha sido pedido, porém, na data da entrega, essas coisas já estavam mais claras, e aí gerava estresse, retrabalho e uma sensação "ruim".

Acho que o segredo para melhorar essas coisas que tu comentou é melhorar a comunicação e os processos. Primeiro, fazer com que o pessoal remoto (seja contratado ou terceirizado) seja "parte do time", participando das reuniões, do processo de pull requests / revisão de código, participando das threads do projeto e etc. Também é legal organizar algumas reuniões de status recorrentes para o acompanhamento, isso é bom pra todo mundo.

Por fim, é fácil culpar quem estar remoto e dizer coisas como "ah, mas eles estão --longe-- do time de desenvolvimento", ou "eles não estão aqui para discutir isso". O difícil é organizar e mudar a cultura da empresa para que essas coisas façam parte do dia a dia.

---

Relacionados para a discussão:

- https://toggl.com/blog/remote-work-infographic/
- https://toggl.com/out-of-office/
  - https://toggl.com/out-of-office-hiring-remote-team/
  - https://toggl.com/out-of-office-remote-company-culture/

**Comment by bernardodiasc** [comment]
> - Os profissionais trabalham durante o dia alocado e atendem as agências a noite/madrugada quando podem?
> - Os profissionais trabalham home office mas se enchem de freelas e não conseguem atender as agências?
> - A falta de responsabilidade é grande o suficiente que o profissional não está nem aí para a entrega do projeto?

O cara que trabalha remoto geralmente oferece algum diferencial que é muito difícil de encontrar no local da empresa. Esse é o fator mais relevante para se contratar um profissional remoto.

Empresas que buscam profissionais remoto para pagar pouco vai achar apenas profissionais completamente atarefados, pois estes precisam fechar a conta pegando tudo que puderem.
